docs/configuration/config-file-reference.md

```diff
@@ -2513,19 +2649,35 @@ api:
 
 The `configstore_config` configures the config database storing rules and alerts, and is used by the Cortex alertmanager. The supported CLI flags `<prefix>` used to reference this config block are:
 
-- `alertmanager`
-- `ruler`
+- `alertmanager.client`
+- `ruler.`
```

Comment:
The ending `.` here means there's a misconfiguration somewhere in the `RegisterFlagsWithPrefix()` which causes a double `..` in the final generated CLI flag.